test/test_cuda.py

```diff
@@ -1542,6 +1542,14 @@ def test_gesv_batched(self):
     def test_gesv_batched_dims(self):
         _TestTorchMixin._test_gesv_batched_dims(self, lambda t: t.cuda())
 
+    @unittest.skipIf(not TEST_MAGMA, "no MAGMA library detected")
+    def test_potrs(self):
+        _TestTorchMixin._test_potrs(self, lambda t: t.cuda())
```

Comment:
I don't think we ever tested for CUDA `potrs` in the past. Added it here.

Reply:
good catch!